packages/webpack/src/runtime/runtime.js

```diff
@@ -150,12 +177,9 @@ const findResourceId = (moduleId) => {
  */
 const wrapRequireWithPolicy = (__webpack_require__, referrerResourceId) =>
   function (specifier) {
-    if (!LAVAMOAT.unenforceable.includes(specifier)) {
-      const requestedResourceId = findResourceId(specifier)
-      enforcePolicy(requestedResourceId, referrerResourceId)
-    }
-    // @ts-ignore - unknown this is the point here
-    return __webpack_require__.apply(this, arguments)
+    // @ts-expect-error - `this` is unknowable
+    const requireThat = __webpack_require__.bind(this, ...arguments)
+    return enforcePolicy(specifier, referrerResourceId, requireThat)
```

Comment:
You should be able to do something like

```js
  /** @this {object} */
  function (specifier) {
    // ...
  }
```

to avoid the `@ts-expect-error`